FlaskApp/app/static/js/script_barchart.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by iris on 2/20/16.
```

Comment:
@hyperchi It's generated automatically, I will remove it since it's not necessary.